api/api.go

```diff
@@ -319,7 +322,16 @@ func (api *Server) GetChainMeta(ctx context.Context, in *iotexapi.GetChainMetaRe
 			GravityChainStartHeight: gravityChainStartHeight,
 		}
 	}
-	return &iotexapi.GetChainMetaResponse{ChainMeta: chainMeta}, nil
+
+	targetHeight := api.bs.TargetHeight()
+	var stage string
+	if targetHeight > tipHeight {
```

Comment:
AFA i understand, we cannot tell the sync status by comparing the TargetHeight() vs tipHeight? b/c once syncer reaches tipHeight, it will always set targetHeight = tipHeight + buf.Size() -- to keep syncing more blocks

would need to measure the *speed* of syncing to tell if tip is reached -- if the rate is faster than 5 second/block it is still syncing, otherwise it has reached tip